packages/camera/camera/lib/src/camera_controller.dart

```diff
@@ -372,9 +367,7 @@ class CameraController extends ValueNotifier<CameraValue> {
     }
     try {
       await CameraPlatform.instance.resumePreview(_cameraId);
-      value = value.copyWith(
-          isPreviewPaused: false,
-          previewPauseOrientation: const Optional<DeviceOrientation>.absent());
```

Comment:
Unfortunately this seems to break camera orientation changes while preview is paused. Since `copyWith` can no longer be told to clear `previewPauseOrientation`, it keeps its stale value and locks the `CameraPreview` widget in the old orientation until the next `pausePreview` call.

A few of the other fields appear to be nullable as well, but I haven't looked closely to see if there are any other regressions.

Reply:
Oh ok, sorry I didn't realize this. I may have to do some refactoring where `Optional<DeviceOrientation>` was used. Thanks for letting me know!